tests/network-setups/fabric/dev/scripts/setupCCLocal.sh

```diff
@@ -18,6 +17,7 @@ fi
 cp -r ${INTEROP_CC_PATH} .
 
 cd fabric-interop-cc
+cp -r $PWD/../../../../../../common/protos-go contracts/interop/
```

Comment:
I think just copying is not enough to  use locally built protos-go. Need to modify `go.mod` as well in interopcc.

Reply:
Fixed the script. Added a `go.mod.local` file for local builds.